Settings.h

```diff
@@ -51,7 +51,8 @@ typedef struct Settings_ {
    int config_version;
    HeaderLayout hLayout;
    MeterColumnSetting* hColumns;
-   Hashtable* dynamicColumns;
+   Hashtable* dynamicColumns; /* runtime-discovered Columns */
+   Hashtable* dynamicMeters;  /* runtime-discovered meters */
```

Comment:
Consistent letter-casing …
```suggestion
   Hashtable* dynamicColumns; /* runtime-discovered columns */
   Hashtable* dynamicMeters;  /* runtime-discovered meters */
```

Reply:
+1, thanks